tests/Makefile

```
@@ -0,0 +1,31 @@
+TAG = edge
+PREFIX = nginx-gateway-fabric
+NGINX_PREFIX = $(PREFIX)/nginx
+PULL_POLICY=Never
+GW_API_VERSION ?= 1.0.0
+K8S_VERSION ?= latest ## Expected format: 1.24 (major.minor) or latest
+
+.PHONY: help
+help: Makefile ## Display this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: create-kind-cluster
+create-kind-cluster: ## Create a kind cluster
+	cd .. && make create-kind-cluster
+
+.PHONY: build-images
+build-images: ## Build NGF and nginx images
```

```suggestion
build-images: ## Build NGF and NGINX images
```

To keep consistency with load-images below